vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -745,7 +747,20 @@ protected Optional<Target> targetByName(String name) {
           .select(WORKFLOW.NAME, WORKFLOW.MAX_IN_FLIGHT)
           .from(WORKFLOW)
           .forEach(record -> maxInFlightPerWorkflow.set(record.value1(), record.value2()));
+
+      DSL.using(connection, SQLDialect.POSTGRES)
+          .select(WORKFLOW.NAME)
+          .from(WORKFLOW)
+          .forEach(
+              record -> DSL.select(WORKFLOW_RUN.HASH_ID,WORKFLOW_RUN.CONSUMABLE_RESOURCES)
+                  .from(
+                      WORKFLOW_RUN
+                          .join(WORKFLOW_VERSION)
+                          .on(WORKFLOW_RUN.WORKFLOW_VERSION_ID.eq(WORKFLOW_VERSION.ID)))
+                  .where(WORKFLOW_VERSION.NAME.eq(record.value1()))
+                  .forEach(record1 -> priorityPerWorkflow.set(record.value1(), record1.value1(), record1.value2())));
```

Comment:
```suggestion
      DSL.using(connection)
          .select(WORKFLOW.NAME, WORKFLOW_RUN.HASH_ID,
              DSL.jsonObject(ACTIVE_WORKFLOW_RUN.CONSUMABLE_RESOURCES))
          .from(WORKFLOW)
          .join(WORKFLOW_VERSION)
          .on(WORKFLOW.NAME.eq(WORKFLOW_VERSION.NAME))
          .join(WORKFLOW_RUN)
          .on(WORKFLOW_RUN.WORKFLOW_VERSION_ID.eq(WORKFLOW_VERSION.ID))
          .join(ACTIVE_WORKFLOW_RUN)
          .on(WORKFLOW_RUN.ID.eq(ACTIVE_WORKFLOW_RUN.ID))
          .where(ACTIVE_WORKFLOW_RUN.ENGINE_PHASE.eq(Phase.WAITING_FOR_RESOURCES))
          .forEach(record -> {
            try {
              priorityPerWorkflow.set(record.value1(),
                  record.value2(),
                  MAPPER.readTree((record.value3() == null || record.value3().data() == null) ?
                      "{}" :
                      record.value3().data()));
            } catch (JsonProcessingException e) {
              // not a disaster; we might just get some things running out of priority
              // until max-in-flight gets saturated
              System.out.println("Uh oh, failed to serialize the consumable resources "
                  + "field on active workflow run for priority by workflow on startup:");
              System.out.println(e.getMessage());
            }
          });
```
This ends up being faster, because it's a single query (rather than query by workflow and then query for matching workflow runs) - and only selects queued workflow runs, which are the only ones that we want to assess for priority.

Note that it's pulling the field `ACTIVE_WORKFLOW_RUN.CONSUMABLE_RESOURCES` - the field is already on `active_workflow_run`, so no need to duplicate it on `workflow_run`. (And since it's only useful for queued workflow runs, we don't need it on ProvenanceWorkflowRun, which is only completed workfow runs)